src/Opdex.Platform.WebApi/Controllers/AuthController.cs

```diff
@@ -53,8 +52,6 @@ public class AuthController : ControllerBase
     /// <response code="204">Signature was validated successfully.</response>
     /// <response code="400">The request is not valid.</response>
```

Comment:
If these aren't providing any benefits, maybe we remove them now.

Reply:
Done 👍